relentless/potential/potential.py

```diff
@@ -584,21 +703,31 @@ def regularize(self, u, f, trim=True):
 
         # find first point from end with sufficient force and cutoff the potential after it
         if self.fcut is not None:
-            flags = np.abs(np.flip(f)) >= self.fcut
-            cut = len(f)-1 - np.argmax(flags)
-            u -= u[cut]
-            if cut < len(f)-1:
-                u[(cut+1):] = 0.
-                f[(cut+1):] = 0.
+            flags = np.abs(np.flip(f)) <= self.fcut
+            cut = len(f) - np.argmax(flags)
+            if cut < len(f):
+                if self._shift:
+                    u -= u[cut]
+                u[cut:] = 0.
+                f[cut:] = 0.
```

Comment:
What errors were you running into that you needed to change this logic so much? I see why you needed to change the indexes so that u is actually 0 _at_ cut, instead of being 0 at the _next_ index. I can't remember why I did it the other way before, but this seems like it should work just fine.

Based on what we discussed, though, I think this should actually be `>` not `<=` for the flags, as I think argmax is always going to return 0 if `<=` is used.